src/components/Routes.js

```diff
@@ -6,11 +6,11 @@ import Dashboard from "../feature/Dashboard";
 
 function AppRoutes() {
   return (
-      <Routes>
-        <Route path="/" element={<Login />} />
-        <Route path="/registration" element={<Registration />} />
-        <Route path="/dashboard" element={<Dashboard />} />
-      </Routes>
+    <Routes>
+      <Route path="/" element={<Registration />} />
+      <Route path="/login" element={<Login />} />
+      <Route path="/dashboard" element={<Dashboard />} />
+    </Routes>
```

Comment:
I believe this has moved to a new file. Correct it

Reply:
@Doyin02 you have not done what is expected. The routes definitions has moved to a separate file, remember.
— This is what you should have done 👇 

```
import AppRoutes from "./components/Routes";
```

```suggestion
   AppRoutes />;
```